cpp/src/Slice/Grammar.y

```diff
@@ -1581,7 +1595,6 @@ throws
 class_export
 // ----------------------------------------------------------------------
 : data_member
-| operation
```

Comment:
I think it would be better to handle the case where operations are declared in classes, and issue an error message saying 'operations aren't allowed in classes' or something. Otherwise the compiler will just emit the default 'syntax error' message, which isn't helpful at all.

Reply:
I agree, let's add this back in another PR.